src/Nethermind/Nethermind.Blockchain/GenesisLoader.cs

```diff
@@ -85,7 +85,7 @@ private void Preallocate(Block genesis)
                 {
                     Transaction constructorTransaction = new SystemTransaction()
                     {
-                        SenderAddress = address,
+                        SenderAddress = genesis.ConstructorSender ?? address,
```

Comment:
Would like to understand the idea of this `ConstructorSender `

Reply:
This was needed for one of their testnets, where deployed contracts in genesis constructors were from Address 0x0...